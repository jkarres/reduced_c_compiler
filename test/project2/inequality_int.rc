
function : void main() {
  int i = 6, j = 7;

  cout << 1 << " " << i < i << endl;
  cout << 2 << " " << i < j << endl;
  cout << 3 << " " << j < i << endl;
  cout << 4 << " " << j < j << endl;

  cout << 5 << " " << i <= i << endl;
  cout << 6 << " " << i <= j << endl;
  cout << 7 << " " << j <= i << endl;
  cout << 8 << " " << j <= j << endl;

  cout << 9 << " " << i > i << endl;
  cout << 10 << " " << i > j << endl;
  cout << 11 << " " << j > i << endl;
  cout << 12 << " " << j > j << endl;

  cout << 13 << " " << i >= i << endl;
  cout << 14 << " " << i >= j << endl;
  cout << 15 << " " << j >= i << endl;
  cout << 16 << " " << j >= j << endl;

}
