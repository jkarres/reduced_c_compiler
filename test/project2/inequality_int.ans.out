1 false
2 true
3 false
4 false
5 true
6 true
7 false
8 true
9 false
10 false
11 true
12 false
13 true
14 false
15 true
16 true
